components/AdminUserDashboard.tsx

```diff
@@ -13,6 +13,12 @@ export default function AdminUserDashboard(): JSX.Element {
     push,
     query: { dashboard }
   } = useRouter()
+  const { API_MANAGER_ENABLED } = process.env
+  const activeUserTypes = USER_TYPES.filter((userType) => {
+    if (userType.value === 'api' && !API_MANAGER_ENABLED) return false
+    return true
```

Comment:
It could be, but I kept it like this [to make this look cleaner ](https://github.com/ibi-group/otp-admin-ui/pull/62/commits/db8d53c7d6c27334f5db9fe16cfc4d0f7d8a82da#diff-0f85c27f67491eec76041639eb70a45fd467de3bd93ff973909132565cd32f14R20)

Reply:
I'd side with @daniel-heppner-ibigroup on this.

Reply:
Would you then suggest making one huge condition in the next branch?

Reply:
Done in 3a29c33ffe0b1276cc11e44d7ce6ffe766f99d71, and we can revisit on the next PR